tests/general/InvalidLinkBearTest.py

```diff
@@ -141,3 +144,29 @@ def test_redirect_threshold(self):
 
         self.assertResult(valid_file=long_url_redirect,
                           invalid_file=short_url_redirect)
+
+    def test_ignore_regex(self):
+
+        ignored_URLs = """
+        http://sub.example.com
+        http://sub.example.com/something
+        """.splitlines()
+
+        not_ignored_URLs = """
+        http://www.example404.com
+        http://exampe.com
+        http://example.co.in
```

Comment:
you didn't add what I asked: `goodexample.com` which might fail if the regex doesn't take care of `example` only being the domain!